wallet/how-to/use-siwe.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Enable your users to Sign-In with Ethereum.
+sidebar_position: 5.5
+---
+
+# Use Sign-In with Ethereum
+
+You can set up [Sign-In with Ethereum (SIWE)](https://docs.login.xyz/) to enable users to easily 
+sign in to your dapp by authenticating with their MetaMask wallet.
+
+MetaMask supports the SIWE standard message format as specified in [ERC-4361](https://eips.ethereum.org/EIPS/eip-4361).
+When your dapp prompts a user to sign a message that follows the SIWE format,
+MetaMask parses the message and gives the user a friendly interface prompting them to sign in to
+your dapp:
+
+<p align="center">
+    <img src={require("../assets/siwe.png").default} alt="Sign-in with Ethereum request" style={{border: '1px solid black'}} />
+</p>
+
+## Domain binding
+
+MetaMask supports domain binding with SIWE to help prevent phishing attacks.
+When a site asks a user to sign a SIWE message, but the domain in the message doesn't match the site
+the user is on, MetaMask displays a warning in the sign-in interface.
+The user must explicitly select to proceed, accepting the risk of a phishing attack.
+
+:::caution important
+MetaMask displays a prominent warning for mismatched domains, but does **not** block users from
+bypassing the warning and accepting the sign-in request.
+This is to not break existing dapps that may have use cases for mismatched domains.
+:::
+
+<div class="row">
+    <div class="column">
+        <img src={require("../assets/siwe-bad-domain.png").default} alt="Sign-in bad domain" style={{border: '1px solid black'}} />
+    </div>
+    <div class="column">
+        <img src={require("../assets/siwe-bad-domain-2.png").default} alt="Sign-in bad domain pop-up" style={{border: '1px solid black'}} />
+    </div>
+</div>
+
+## Example
+
+The following is an example of setting up SIWE with MetaMask using
+[`personal_sign`](https://metamask.github.io/api-playground/api-documentation/#personal_sign).
+See the [live example](https://metamask.github.io/test-dapp/#siwe) and
+[test dapp source code](https://github.com/MetaMask/test-dapp).
+
+<!--tabs-->
+
+# JavaScript
+
+```javascript
+const siweSign = async (siweMessage) => {
+  try {
+    const from = accounts[0];
+    const msg = `0x${Buffer.from(siweMessage, 'utf8').toString('hex')}`;
+    const sign = await ethereum.request({
+      method: 'personal_sign',
+      params: [msg, from, 'Example password'],
```

Comment:
Should we have some information/details about what the "Example password" is, and what it's used for? I myself have always been confused about it, and I don't see it explained anywhere.

Reply:
@alexandratran turns out that there's no "example password" parameter. can we remove it everywhere as part of this PR as well?